package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.3.5",
   "description": "AST utility module for statically analyzing JSX",
   "main": "lib/index.js",
+  "files": [
+    "lib"
```

Comment:
I'm not proposing running a full test suite on your published output, just that your module is requireable. I realize that wasn't clear, sorry. I think requireability is the main issue if you think you didn't package it correctly (and mostly covers your last point) or some build step generated bad JS (eg babel somehow screws up or some gulp template is bad).

`npm test` isn't going to test your actual packaged code if you're compiling, so I see little value in doing that, and like you said it would be pretty arduous to try.

Here's what I would do (I'd consider putting it in `prepublish` too, though I wish that didn't run on `npm install`):

```sh
npm pack
cd /tmp
npm install <path to pkg.tgz>
node -e "require('pkg'); require('pkg/other/entry'); " && echo "SUCCESS"
```

Anyway. Like I said, agree to disagree here. It's a super small detail and in the grand scheme, incredibly unimportant.